ConsoleApp1/Engin.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Engin
+    {
+        private string _enginType;
+
+        Engin(string enginType)
+        {
+            _enginType = enginType;
+        }
+
+        public string EnginType 
```

Comment:
if you need only getter you can simplify it to the `public string EnginType => _engineType;`